docs/source/user_guide/plots.rst

```diff
@@ -0,0 +1,969 @@
+|Dataset| and then attach it to a |Session|.
+
+The example below demonstrates setting up an interactive location scatterplot
+for the :ref:`quickstart-geo <dataset-zoo-quickstart-geo>` dataset that is
+:ref:`attached to the App <attaching-plots>`.
+
+In this setup, the location plot renders each sample using its corresponding
+``[longitude, latitude]`` coordinates from the dataset's only |GeoLocation|
+field, ``location``. When points are lasso-ed in the plot, the corresponding
+samples are automatically selected in the session's current
+:meth:`view <fiftyone.core.session.Session.view>`. Likewise, whenever you
+modify the Session's view, either in the App or by programmatically setting
+:meth:`session.view <fiftyone.core.session.Session.view>`, the corresponding
+locations will be selected in the scatterplot.
+
+Each block in the example code below denotes a separate cell in a Jupyter
+notebook:
+
+.. code-block:: python
+    :linenos:
+
+    import fiftyone as fo
+    import fiftyone.brain as fob
+    import fiftyone.zoo as foz
+
+    dataset = foz.load_zoo_dataset("quickstart-geo")
+
+    # Index the dataset by visual uniqueness
+    fob.compute_uniqueness(dataset)
+
+    # Launch the App
+    session = fo.launch_app(dataset)
+
+.. code-block:: python
+    :linenos:
+
+    from fiftyone import ViewField as F
+
+    # The number of ground truth objects in each sample
+    num_objects = dataset.values("ground_truth", F("detections").length())
+
+    # Create the scatterplot
+    plot = fo.location_scatterplot(
+        samples=dataset,
+        labels="uniqueness",    # color points by their `uniqueness` values
+        sizes=num_objects,      # scale point sizes by number of objects
+    )
+    plot.show(height=720)
+
+    # Connect to session
+    session.plots.attach(plot)
+
+.. image:: ../images/plots/location-scatterplot-interactive.gif
+   :alt: location-scatterplot-interactive
+   :align: center
+
+.. _confusion-matrices:
+
+Confusion matrices
+__________________
+
+When you use evaluation methods such as
+:meth:`evaluate_classifications() <fiftyone.core.collections.SampleCollection.evaluate_classifications>`
+and
+:meth:`evaluate_detections() <fiftyone.core.collections.SampleCollection.evaluate_detections>`
+to evaluate model predictions, the confusion matrices that you can generate
+by calling the
+:meth:`plot_confusion_matrix() <fiftyone.utils.eval.classification.ClassificationResults.plot_confusion_matrix>`
+method are responsive plots that can be attached to App instances to
+interactively explore specific cases of your model's performance.
+
+.. note::
+
+    See :ref:`this page <evaluating-models>` for an in-depth guide to using
+    FiftyOne to evaluate models.
+
+The example below demonstrates setting up an interactive confusion matrix for
+the results of evaluating the predictions in the ``predictions`` field of the
+:ref:`quickstart <dataset-zoo-quickstart>` dataset.
```

Comment:
Phrasing:
"setting up an interactive confusion matrix for the results of an evaluation on the ``predictions`` field of the ..."